recipe/meta.yaml

```diff
@@ -24,8 +24,6 @@ test:
   imports:
     - billiard
     - billiard.dummy
-    - billiard.tests
-    - funtests
```

Comment:
FYI they dropped these modules from the install. So have dropped them here. Things have been restructured into a package called [`t`]( https://github.com/celery/billiard/tree/v3.5.0.1/t ), which is [excluded]( https://github.com/celery/billiard/blob/v3.5.0.1/setup.py#L229 ) from the install.